azure-fluid-relay-preview-pr/articles/howtos/hosting.md

```diff
@@ -0,0 +1,240 @@
+---
+title: Hosting Fluid Example
+description: Detailed explanation about how Fluid examples can be hosted on Azure App Service and Static Web App
+---
+
+This article demonstrates how to deploy your Fluid examples to Azure App Service and Static Web Apps. [FluidHelloWorld](https://github.com/microsoft/FluidHelloWorld/tree/main-frs) repository is a Fluid example which contains a simple app that enables all connected clients to roll a dice and view the result. 
+
+# Pre-requisites
+1. Add the following extensions to Visual Studio Code: 
+    - [Azure Account](https://marketplace.visualstudio.com/items?itemName=ms-vscode.azure-account): It is required to sign in to your `Azure Subscription`.
+
+    You can install any one of the following extensions depending on the type of deployment you choose:
+
+    - [Azure App Service](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azureappservice): It is required for deploying app using `App Services`.
+    - [Azure Static Web App](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurestaticwebapps): It is required for deploying app using `Static Web Apps`.
+
+These extensions allow the developer to deploy to Azure directly from Visual Studio Code.
+
+2. Run the following command to clone the repository:
+
+```shell
+git clone https://github.com/microsoft/FluidHelloWorld
+```
+
+# Connecting to Azure Fluid Relay Service
+You can connect to `Azure Fluid Relay service` by providing the tenant ID and key that is uniquely generated for you when onboarding to the service. Although you can build your own implementation that fulfills the `ITokenProvider` interface, we provide two in-built options for instantiating a `tokenProvider`:
+
+1. `InsecureTokenProvider`: This type of implementation is only recommended for development purposes as it exposes the tenant key secret in your client-side code bundle. You can add the below `config` in [app.ts](https://github.com/microsoft/FluidHelloWorld/blob/main/src/app.ts), replacing the strings with the values appropriate for your assigned instance.
+
+```typescript
+import { AzureClient, AzureConnectionConfig } from "@fluidframework/azure-client";
+import { InsecureTokenProvider } from `@fluidframework/test-client-utils`;
+
+const config: AzureConnectionConfig = {
```

Comment:
The HelloWorld repo is written in JS, not TS, so copying over this codeblock will throw an error

![image](https://user-images.githubusercontent.com/1434956/135646623-6d172fab-c67b-4424-babc-c445377a6228.png)